packages/orbit-components/src/Modal/README.md

```diff
@@ -39,6 +39,7 @@ Table below contains all types of the props available in the Modal component.
 | disableAnimation    | `boolean`                  | `false`    | Defines whether the Modal performs the slide in animation on mobile. If you want to improve your [CLS](https://web.dev/cls/) score, you might want to set this to `true`.                            |
 | mobileHeader        | `boolean`                  | `true`     | If `false` the ModalHeader will not have MobileHeader and CloseContainer                                                                                                                             |
 | labelClose          | `React.Node`               | `Close`    | The label for the close button.                                                                                                                                                                      |
+| onScroll            | `event => void \| Promise` |            | Function for handling onScroll event. If you don't pass any function the onScroll event will not be handled. [See Functional specs](#functional-specs)                                               |
```

Comment:
I would remove this sentence. It gives the feeling that if you don't pass `onScroll`, scrolling won't be possible. But that is not true, right?
```suggestion
| onScroll            | `event => void \| Promise` |            | Function for handling onScroll event. [See Functional specs](#functional-specs).                                              |
```

Reply:
yeah, it's a bit confusing, should be callback instead of event, but I will remove that sentence completely 👍🏻